css-containment/Overview.bs

```diff
@@ -138,6 +138,37 @@ Layout Containment</h3>
 
 	2. The element must be a <a>formatting context</a>.
 
+	3. If a <a>fragmentation context</a> participates in <a>layout containment</a>,
+		the first element with <a>layout containment</a>
+		affecting the <a>fragmentation context</a>
+		must “trap” the remainder of the <a>fragmented flow</a>.
+		<a>Fragmentation</a> must not continue past the <a>layout containment</a> boundary,
+		and the last <a>fragmentation container</a>
+		within the first <a>layout containment</a> boundary
+		is treated as if it is the last <a>fragmentation container</a>
+		in its <a>fragmentation context</a>.
+
+		If subsequent <a>fragmentation containers</a> in the <a>fragmentation context</a>
+		are only generated when more content remains in the <a>fragmented flow</a>,
+		then they are not generated.
+		If they would exist regardless,
+		they remain part of the <a>fragmentation context</a>,
+		but do no receive any content from the <a>fragmented flow</a>.
+
+		Specifically:
+			- <a spec=css-regions>CSS Regions</a> following the one which traps the content
```

Comment:
Oops, I actually knew that one, but assumed based on file extension .src.html that css-regions was not under bikeshed, and then I didn't know. But it is, so I'll fix this. How often is the database refreshed once I've done this?